crates/ty_python_semantic/src/types/ide_support.rs

```diff
@@ -119,13 +135,9 @@ impl<'db> AllMembers<'db> {
             ),
 
             Type::NominalInstance(instance) => {
-                let class_literal = instance.class_literal(db);
+                let (class_literal, specialization) = instance.class(db).class_literal(db);
                 self.extend_with_instance_members(db, ty, class_literal);
-
-                // If this is a NamedTuple instance, include members from NamedTupleFallback
-                if CodeGeneratorKind::NamedTuple.matches(db, class_literal, None) {
-                    self.extend_with_type(db, KnownClass::NamedTupleFallback.to_class_literal(db));
```

Comment:
Looks like this was wrong before, because we were extending an instance type with class members. Not a huge deal, but leads to wrong types (function literals instead of bound methods). Fixed here.

Reply:
I guess we could add a test to completions.rs that shows that we now display the type correctly for the suggestions? Not essential though

Reply:
Decided that this is not important enough.